src/integration/billing/stripe/StripeBillingIntegration.ts

```diff
@@ -1078,16 +1078,21 @@ export default class StripeBillingIntegration extends BillingIntegration {
       startDate,
       startTime,
       timeSpent,
-      totalConsumption: consumptionkWh,
+      totalConsumption: formattedConsumptionkWh,
       siteAreaName: chargeBox?.siteArea?.name,
       chargeBoxID: transaction?.chargeBoxID,
       stopTime,
     });
     return description;
   }
 
-  private convertConsumptionToKWh(transaction: Transaction): number {
-    return new Decimal(transaction.stop.totalConsumptionWh).dividedBy(10).round().dividedBy(100).toNumber();
+  private formatConsumptionToKWh(transaction: Transaction): string {
+    // ACHTUNG: consumed energy shown in the line item might be slightly different from the billed energy
+    return new Decimal(transaction.stop.totalConsumptionWh).dividedBy(1000).toNumber().toLocaleString(this.getUserLocale(transaction));
```

Comment:
There's an helper to create Decimal in `Utils.ts`. There's also helpers to round them given the value type they represent.

Reply:
All Decimal creation handled in one code place is beneficial ;)